### PR TITLE
Headings page and updates to sidebar nav

### PR DESCRIPTION
**Pull request by ildrinn**
Created a new Accessibility Headings page, and updated the sidebar nav to include this

### CODE REVIEW

**Review by MarionDownie — CHANGES_REQUESTED** [review]
Hi Susi, sorry for the delay. Can you please just a few small changes:

- typo in table, rouge 'C'- 'CStruggles with pages that have dense, unstructured or a lot of text'
- is it possible to remove the bullets before the staggered headings? Would look neater
- check spacing - some of the headings are very close to the text above. Perhaps just add in paragraph tags/breaks
- screenreader is shown as two words in the penultimate para - can you please make this one word, as elsewhere?
- link at the end is broken

**Comment by ildrinn** [comment]
Hi Marion - I think I've done this now? (made the changes and pushed the updated file). Still learning GitHub so let me know if I've missed anything.

**Review by RobinKnipe — CHANGES_REQUESTED** [review]
Just a couple of minor amendments.

File: apps/docs/src/common/pages/accessibility/headings.tsx [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+<p>An example of a nested heading structure: </p>
+<img src={headingImage} alt="Headings nested in a sequential hierarchy"  width="50%"/>
+
+<h3>Structure is separate from appearance</h3>
+
+<p>Don’t use heading tags for non-heading purposes. For example, if you need a call-out message in large text but it does not function as a heading, then use styling to achieve this visual effect. </p>
+
+<p>As a rule of thumb: if there is nothing or very little that sits underneath to it, it’s probably not a heading. </p>
+
+<p>Similarly, if part of your content works as a heading (for example, to head up content blocks in the footer, or in a sidebar list of related links) then you must use the correct html code as well as any visual styling. If you don’t, a screen reader will not recognise this as a heading and the user may miss it.</p>
```
[the remark this answers is not in the document]
Comment (daniel-ac-martin) [reply apps/docs/src/common/pages/accessibility/headings.tsx]: Should be fixed.

File: apps/docs/src/common/pages/accessibility/headings.tsx [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+            <li>&lt;h4&gt;</li>
+          </ul>
+        </li>
+      </ul>
+    </li>
+  </ul>
+</li>
+</ul>
+
+<p>Never skip heading levels – if you need a heading to look different, do this with css styles rather than with the html code. </p>
```
[the remark this answers is not in the document]
Comment (daniel-ac-martin) [reply apps/docs/src/common/pages/accessibility/headings.tsx]: Should be fixed.

**Comment by daniel-ac-martin** [comment]
@MarionDownie, @RobinKnipe: Ready to merge?

**Comment by MarionDownie** [comment]
> Hi Marion - I think I've done this now? (made the changes and pushed the updated file). Still learning GitHub so let me know if I've missed anything.

Hi Susi, sorry for the delay - I've been scratching my head as to why I can't see your changes, but Daniel says they haven't been pushed. Can you please check and try again? Guess you may need to pull the latest from master.

File: apps/docs/src/common/pages/accessibility/headings.tsx [hunk 3]
```diff
@@ -137,39 +137,39 @@ const Page: FC<PageProps> = ({ location }) => (
...
+<ul class="govuk-list">
 <li>&lt;h1&gt;
-  <ul>
+  <ul class="govuk-list">
     <li>&lt;h2&gt;
-      <ul>
+      <ul class="govuk-list">
         <li>&lt;h3&gt;
-          <ul>
+          <ul class="govuk-list">
```
Comment (daniel-ac-martin) [inline apps/docs/src/common/pages/accessibility/headings.tsx]: These all need to be `className=` rather than `class=`.

**Review by daniel-ac-martin — CHANGES_REQUESTED** [review]

**Review by MarionDownie — COMMENTED** [review]
Looks good - just two small things. 

Is it possible to keep the style of the nested headings Susi had previously without the bullets? I think the indents were useful to convey the structure visually (but it's better with nothing than with the extra bullets)

Could you add an extra space underneath the image and before the heading 'Structure is separate from appearance'?

**Comment by daniel-ac-martin** [comment]
@haggishunt56: Would you mind making the two small amendments that Marion asked for above?

**Comment by daniel-ac-martin** [comment]
With the 'nested headings', I think she wants something like:
```
<h1>
  <h2>
    <h3>
      <h4>
```

**Comment by haggishunt56** [comment]
> With the 'nested headings', I think she wants something like:
> 
> ```
> <h1>
>   <h2>
>     <h3>
>       <h4>
> ```

I've made this change, but think it would make more sense to the reader to close each tag too (and possibly include tags all the way up to `<h6>`). What do you think @daniel-ac-martin & @MarionDownie?

**Comment by daniel-ac-martin** [comment]
@haggishunt56: Let's go with this for now.
We can always make further enhancements later.

**Pull request merged**